### PR TITLE
Add new word students to each line of output

### PR DESCRIPTION
**Pull request by 918benjamin**
Some comments here about why I did this commit / feature

### CODE REVIEW

**Comment by jakedevar** [comment]
Wow this is great just change a bit!

File: app.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
+let word = 'students';
```
Comment (918benjamin) [inline app.js:1]: We actually need to be using "newWord" here (related to issue #7 )

**Comment by jakedevar** [comment]
#7 make sure you syntax is correct

**Comment by jakedevar** [comment]
Changes look good merging now

**Pull request merged**